# HG changeset patch
# User Sean Farley <sean@mcs.anl.gov>
# Date 1332269669 18000
#      Tue Mar 20 13:54:29 2012 -0500
# Node ID 88049269953c67c3fdcc4309bf901508a875f0dc
# Parent  aed7f236f05776012de66d24f7abf7b0395e7a97
cmake: added install_name for shared prefix builds

diff -r aed7f236f057 -r 88049269953c CMakeLists.txt
--- a/CMakeLists.txt	Tue Mar 20 13:54:29 2012 -0500
+++ b/CMakeLists.txt	Tue Mar 20 13:54:29 2012 -0500
@@ -11,10 +11,11 @@ else()
 endif()
 
 # Configure libmetis library.
 if(SHARED)
   set(METIS_LIBRARY_TYPE SHARED)
+  set(CMAKE_INSTALL_NAME_DIR "${CMAKE_INSTALL_PREFIX}/lib")
 else()
   set(METIS_LIBRARY_TYPE STATIC)
 endif(SHARED)
 
 include(${GKLIB_PATH}/GKlibSystem.cmake)
